src/applications/caregivers/actions/fetchMapBoxGeocoding.js

```diff
@@ -4,7 +4,7 @@ import mapboxClient from '../utils/mapbox/mapboxClient';
 import { replaceStrValues } from '../utils/helpers';
 import content from '../locales/en/content.json';
 
-export const fetchMapBoxBBoxCoordinates = async (query, client = null) => {
+export const fetchMapBoxGeocoding = async (query, client = null) => {
```

Comment:
Renamed this to be more generic since it returns a lot of helpful data. We are currently only using the lat/long returned from this call.